homeassistant/components/enocean/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "EnOcean",
+  "config": {
+    "flow_title": "ENOcean setup",
+    "step": {
+      "init": {
```

Comment:
Add all the steps with forms and the specific form data names.

Reply:
strings now contains two steps: detect and manual